openpype/hosts/maya/api/plugin.py

```diff
@@ -262,7 +262,12 @@ def _default_remove_instances(self, instances):
         """
         for instance in instances:
             node = instance.data.get("instance_node")
+
             if node:
+                # Some nodes require to be remove from the sets before deleting
+                # the set, else the node will get deleted as well.
+                # For example timeSliderBookmark nodes.
+                cmds.sets(cmds.sets(node, query=True), remove=node)
```

Comment:
This will error if the set was empty to begin with.
Also, you're much better of doing `clear` on the sets.
```suggestion
                # Some nodes require to be removed from the set before deleting
                # the set, else the node will get deleted as well.
                # For example timeSliderBookmark nodes.
                cmds.sets(clear=node)
```